### DISCUSSION TITLE
Reading data from a table [Oct 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6676] [Discord Threads]
> 🙋‍♂️ **iqbalpb**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### curl --request GET \   
####   --url 'http://localhost:8080/api/v1/db/data/<Organisation>/<ProjectID>/<TableId>/' \   
####   --header 'xc-auth: <TOKEN>'   
####    
#### Response will be paginated by default to 25 records. You can increase by including `limit`. For linked records, this will include count only.   
####    
#### Further, you can use below CURL to read further information about a link record   
####    
#### curl --request GET \   
####   --url 'http://localhost:8080/api/v1/db/data/<Organisation>/<ProjectID>/<TableId>/<RowId>/<RelationType>/<RelationalColumnID>' \   
####   --header 'xc-auth: <TOKEN>'   
####    
#### Note:   
#### <Organisation> : defaults to 'noco'   
#### <RelationType>: can be 'mm', 'hm' or 'bt'  
#  
> 💁 **o1lab**  
#### getting data without limit is not a usecase  
#  
> 💁 **o1lab**  
#### for us  
#  
> 🙋‍♂️ **iqbalpb**  
#### <@857998597474484295>  thanks I tried by setting limits from count API endpoint and my issue solved and <@310157751395287040> thanks for your reply  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1147793913571196978)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
